corgidrp/darks.py

```diff
@@ -249,9 +249,13 @@ def build_trad_dark(dataset, detector_params, detector_regions=None, full_frame=
     exthdr['DATATYPE'] = 'Dark'
 
     master_dark = Dark(data, prihdr, exthdr, dataset, err, dq, errhdr)
+    master_dark.ext_hdr['DRPNFILE'] = int(np.round(np.nanmean(unmasked_num)))
```

Comment:
can you run zz_dataformat_e2e and update the data format doc in docs/source/data_format?